R/extract_param.R

```diff
@@ -1,151 +1,268 @@
 #' Extract parameters
 #'
 #' Extract parameters from summary data (e.g. in meta analysis)
-#' @param type A \code{character} defining whether summary statistics based around \code{percentiles} (default) or \code{range}
-#' @param values A \code{vector}. If \code{type=percentiles}: c(percentile_1, percentile_2); and if \code{type=range}: c(median, min, max)
-#' @param distribution A \code{character} specifying distribution to use. Default is \code{lnorm}; also takes \code{gamma} and \code{weibull}.
-#' @param percentiles A \code{vector} with two elements specifying the percentiles defined in \code{values} if using \code{type="percentiles"}.
-#' @param samples A \code{numeric} specifying the sample size if using \code{type="range"}.
+#' @param type A \code{character} defining whether summary statistics based
```

Comment:
Same comment re doc as for epidist, now updated in [this issue](https://github.com/epiverse-trace/epiparameter/issues/25).